src/main/scala/com/microsoft/hyperspace/actions/RefreshIncrementalAction.scala

```diff
@@ -143,4 +144,28 @@ class RefreshIncrementalAction(
       entry
     }
   }
+
+  override protected def event(appInfo: AppInfo, message: String): HyperspaceEvent = {
+    RefreshIncrementalActionEvent(appInfo, logEntry.asInstanceOf[IndexLogEntry], message)
+  }
+
+  private def resolveConfigIndexedColumns(
```

Comment:
I think we could move this function (resolveConfig in CreateActionBase, isValidIndexSchema in CreateAction) to `object IndexConfig` to avoid duplicate code.

Reply:
No need. I removed it altogether.